### ISSUE TITLE
Database

### ISSUE THREAD

**Issue by SVG490** [Pink-Hat-Hacker/allergen-ai#1]
Could you provide the firebase database jason file because proable allergens is not showing for me

**Comment by sanjeethm** [comment]
> Could you provide the firebase database jason file because proable allergens is not showing for me

Did you get the firebase db json file ?  I am also looking for the data.  Could you please help to share if you have ?

**Comment by Pink-Hat-Hacker** [comment]
Hi everyone, thank you for raising this issue! I used the free trial for LogMeal API which provides the food ingredients so that's why your probable allergens may not have been showing up. Also my database is kind of embarrassing, haha, it should be better written than this. Hope this helps!

That being said I have added the file here: src/allergen-ai-default-rtdb-export.json
[Github Link](https://github.com/Pink-Hat-Hacker/allergen-ai/blob/main/src/allergen-ai-default-rtdb-export.json)

Here is an attached file:
[allergen-ai-default-rtdb-export.json](https://github.com/Pink-Hat-Hacker/allergen-ai/files/15052035/allergen-ai-default-rtdb-export.json)
